src/app/templates/app/login.html

```diff
@@ -36,16 +36,17 @@
   </div>
 </form>
 <h6> Not a user? <a href="/app/register/">Sign up</a> now!</h6>
+<h6><a href="/app/password_reset/">Forgot password ?</a></h6>
```

Comment:
`Forgot Password?` No space between `d` and `?`.
I know it's not a big thing but still UI matters. 😄